docs/sprite-mask.zh-CN.md

```diff
@@ -15,24 +15,34 @@ type: 组件
 3、创建 [SpriteMask](${api}core/SpriteRenderer)
 
 ```typescript
-import { AssetType, Sprite, SpriteMask, Texture2D } from "oasis-engine";
-
-engine.resourceManager
-  .load<Texture2D>({
-    url: "https://gw.alipayobjects.com/mdn/rms_7c464e/afts/img/A*qyhFT5Un5AgAAAAAAAAAAAAAARQnAQ",
-    type: AssetType.Texture2D
-  })
-  .then((texture) => {
-    const spriteEntity = rootEntity.createChild(`spriteMask`);
-    // 给实体添加 SpriteMask 组件
-    const spriteMask = spriteEntity.addComponent(SpriteMask);
-    // 通过 texture 创建 sprite 对象
-    const sprite = new Sprite(engine, texture);
-    // 设置 sprite
-    spriteMask.sprite = sprite;
-  });
+// 创建一个遮罩实体
+const spriteEntity = rootEntity.createChild(`spriteMask`);
+// 给实体添加 SpriteMask 组件
+const spriteMask = spriteEntity.addComponent(SpriteMask);
+// 通过 texture 创建 sprite 对象
+const sprite = new Sprite(engine, texture);
+// 设置 sprite
+spriteMask.sprite = sprite;
+// 设置当前 mask 会对处于哪些遮罩层的精灵有影响
+spriteMask.influenceLayers = SpriteMaskLayer.Layer0;
+
+// 设置遮罩类型
+spriteRenderer.maskInteraction = SpriteMaskInteraction.VisibleInsideMask;
+// 设置精灵处于哪个遮罩层
+spriteRenderer.maskLayer = SpriteMaskLayer.Layer0;
+
 ```
 
+## 遮罩层 (SpriteMaskLayer)
+
+[SpriteMaskLayer](${api}core/SpriteMaskLayer) 里面声明了引擎提供的遮罩层，一共声明了 32 个遮罩层，分别是 Layer0~Layer31，遮罩层和渲染无关，只是为了帮助开发者设置 SpriteMask 和 SpriteRenderer 如何进行关联，一个 SpriteMask 对象要对一个 SpriteRenderer 对象产生遮罩作用的一个前提就是两者的遮罩层有交集。
+
+SpriteMask 的 influenceLayers 表示该 mask 对处于哪些遮罩层内的 SpriteRenderer 会起到遮罩作用，SpriteRenderer 的 maskLayer 表示该精灵处于哪些遮罩层，如下：
```

Comment:
Code like `influenceLayers` should be wrapped with ``

Reply:
> Code like `influenceLayers` should be wrapped with ``

+1